### PR TITLE
 modified file module _2

### PR DESCRIPTION
**Pull request by Avetiq30**

### CODE REVIEW

File: src/file/file.service.ts [hunk 1]
```diff
@@ -24,12 +25,9 @@ export class FileService {
   }
 
   deleteFileById(id: string) {
```
Comment (VahagShahinyan) [inline src/file/file.service.ts:27]: Add async before method

File: src/file/file.service.ts [hunk 2]
```diff
@@ -24,12 +25,9 @@ export class FileService {
...
-    // check if file used in products
-    // delete file from mongodb
-    // delete file from upload dir
     const deletedFile = this.fileModel.findByIdAndDelete(id);
```
Comment (VahagShahinyan) [inline src/file/file.service.ts:28]: Add ```await```

File: src/file/file.service.ts
Comment (VahagShahinyan) [inline src/file/file.service.ts:1]: Add ```async``` for all method of service

File: src/file/file.controller.ts
Comment (VahagShahinyan) [inline src/file/file.controller.ts:1]: Add ```async```  and ```await``` for all method of controller and add ```try catch``` if it is needed

File: src/file/file.service.ts (hunk 1 above)
Comment (Avetiq30) [reply src/file/file.service.ts:27]: ++

File: src/file/file.service.ts (hunk 2 above)
Comment (Avetiq30) [reply src/file/file.service.ts:28]: ++

File: src/file/file.service.ts
Comment (Avetiq30) [reply src/file/file.service.ts:1]: ++

File: src/file/file.controller.ts
Comment (Avetiq30) [reply src/file/file.controller.ts:1]: ++

**Pull request merged**